reports/tasks.py

```diff
@@ -0,0 +1,89 @@
+"""
+Worker tasks for reporting
+"""
+
+# Imports #####################################################################
+
+from datetime import datetime, timedelta
+import logging
+
+from django.conf import settings
+from django.core.mail import EmailMessage
+from django.core.management import call_command
+from huey.api import crontab
+from huey.contrib.djhuey import db_periodic_task
+
+from instance.models.openedx_instance import OpenEdXInstance
+
+
+# Logging #####################################################################
+
+logger = logging.getLogger(__name__)
+
+
+# Constants ###################################################################
+
+TRIAL_INSTANCE_REPORT_SENDER = getattr(settings, 'TRIAL_INSTANCE_REPORT_SENDER',
```

Comment:
@pcockwell, this is not defined in `settings.py` at all and hence will never get overridden. I'd recommend dropping this and just using `settings.DEFAULT_FROM_EMAIL`.